modules/foreman_organization.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2016, Eric D Helms <ericdhelms@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: foreman_organization
+short_description: Manage Foreman Organization
+description:
+    - Manage Foreman Organization
+author: "Eric D Helms (@ehelms)"
+requirements:
+    - "nailgun >= 0.28.0"
+    - "python >= 2.6"
+options:
+    server_url:
+        description:
+            - URL of Foreman server
+        required: true
+    username:
+        description:
+            - Username on Foreman server
+        required: true
+    password:
+        description:
+            - Password for user accessing Foreman server
+        required: true
+    verify_ssl:
+        description:
+            - Verify SSL of the Foreman server
+        required: false
+    name:
+        description:
+            - Name of the Foreman organization
+        required: true
+'''
+
+EXAMPLES = '''
+- name: "Create CI Organization"
```

Comment:
btw, I prefer to write `delefate_to: localhost` instead of `local_action`, both do exactly the same.
```yaml
- name: "Create CI Organization"
  foreman_organization:
      username: "admin"
      password: "admin"
      server_url: "https://fakeserver.com"
      name: "My Cool New Organization"
  delegate_to: localhost
```